src/main/java/com/revature/models/SwaggerDocutest.java

```diff
@@ -0,0 +1,20 @@
+package com.revature.models;
+
+import java.util.List;
+
+import com.revature.services.OASService;
+import io.swagger.models.Swagger;
+import lombok.Data;
+
+@Data
+public class SwaggerDocutest implements Docutest {
+    int id; // id of swaggersummary object
+    private List<Request> requests;
+    
+    // TODO autowire once Spring breans are set up
```

Comment:
Spring Beans are as easy to set up as adding corresponding Stereotype annotations, so if this needs to be autowired, go ahead and setup the beans that are needed. Note that this class would additionally need to be a Spring Bean in order to have its dependencies be autowired.

Reply:
I decided to not inject the service, since it was causing more problems than it was worth. I wanted to have a constructor that takes in a Swagger object as an input, but then Spring would need that Swagger object to also be a Spring bean, which isn't what we want. 

If I use either constructor or setter injection for the OASService, the constructor that takes in the Swagger object does not have the service injected (i.e. adapter is null). Field injection with @Autowired doesn't seem to work either, although I'm not sure why. I originally thought it was because Spring was using that constructor, but having a separate annotated constructor ended up producing the same result. 

None of the following work:
```Java
private OASService adapter;

public Docutest(Swagger input) {
...
}

@Autowired
public setAdapter(OASService adapter) {
...
}
```

```Java
@Autowired
public Docutest(OASService adapter) { 
... 
}

public Docutest(Swagger input) {
...
}
```

```Java
@Autowired
private OASService adapter

public Docutest(Swagger input) {
...
}
```

```Java
@Autowired
private OASService adapter

@Autowired
public Docutest(OASService adapter) {
...
}
public Docutest(Swagger input) {
...
}
```

Reply:
I would recommend removing the custom constructor, and allowing Lombok to generate a Constructor that accepts just the list of requests (or we can create that constructor ourselves if Lombok is inconvenient). Then externally, we can create logic that will use the OASService to obtain the requests. This will allow the model class to not include any logic, which means it can be safely excluded from analysis reports. And the logic for obtaining the requests can be tested and written independently.